zubhub_frontend/zubhub/src/views/search_results/SearchResults.jsx

```diff
@@ -282,6 +330,9 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => {
   return {
+    getStaffPicks: args => {
```

Comment:
I see that we are only getting staff picks, but even when staff pics are enabled, we are not rendering. What could be the issue? Can you test locally and make changes if needed?